packages/yoga/src/Heading/web/__snapshots__/Heading.test.jsx.snap

```diff
@@ -117,10 +117,11 @@ exports[`<Heading /> should match snapshot no padding 1`] = `
 .c5 {
   margin: 0;
   padding: 0;
-  font-size: 14px;
+  font-size: 16px;
   font-weight: 500;
   font-family: Rubik;
   color: #231B22;
+  font-size: 14px;
```

Comment:
@oharaju, first of all great contribution! 🎉
Looking at this snapshot test it looks like that when the `size` prop at the test file was being used the `font-size` on the style was set to `14px` directly but using `font-size` the style looks like to be setting it to `16px` and after being updated to `14px`.
Could we check this behaviour? Another fact that I checked is that any mobile snapshot changed the style after the change from `size` your `fontSize` it seems to work just the same.